lux/vislib/matplotlib/ScatterChart.py

```diff
@@ -57,14 +57,14 @@ def initialize_chart(self):
 
         color_attr = self.vis.get_attr_by_channel("color")
         if len(color_attr) == 1:
-            self.fig, self.ax = matplotlib_setup(6, 5)
             color_attr_name = color_attr[0].attribute
             color_attr_type = color_attr[0].data_type
             colors = df[color_attr_name].values
             plot_code += f"colors = df['{color_attr_name}'].values\n"
             unique = list(set(colors))
             vals = [unique.index(i) for i in colors]
             if color_attr_type == "quantitative":
+                self.fig, self.ax = matplotlib_setup(7, 5)
```

Comment:
The code inside matplotlib_setup doesn't show up when we export the chart. As a result, the figure size in the exported code and the widget is different. Is this the intended behavior?

Reply:
fixed